### PR TITLE
enable viewing individual review

### PR DESCRIPTION
**Pull request by AnnaFrancesD**
Thinking of adding another api request to get the user and display their avatar next to their username

### CODE REVIEW

**Comment by AnnaFrancesD** [comment]
I still need to format the category string as well

File: src/components/review-list.jsx [hunk 1]
```diff
@@ -20,16 +20,22 @@ export default function ReviewList () {
...
         <>
         {isLoading ? (<p>Loading...</p>) : (
             reviews.map((review) => {
-               return <div key={review.review_id} className="preview_review_card">
+               return <div key={review.review_id} className="preview-review-card">
                    <p><b>{review.title}</b></p>
                    <img src={review.review_img_url} alt={review.title} width="100"></img>
                    <p>{review.review_body.slice(0, 71)}...</p>
-                   <button type="button">Read</button>
+                   <button onClick={() => {viewReview(review.review_id)}} type="button">Read 👀</button>
```
Comment (Dutton1990) [inline src/components/review-list.jsx:38]: You could use a Link tag here, I think inside a button it has to be wrapped around the text part only? I feel like you may have tried it then resorted to useNavigate instead. To be honest it works either way so upto you!

File: src/components/review-list.jsx [hunk 2]
```diff
@@ -20,16 +20,22 @@ export default function ReviewList () {
             setIsLoading(false)
         })
     }, [])
+    
+    let navigate = useNavigate()
+    function viewReview (review_id) {
+        setIsLoading(true);
+        navigate(`/reviews/:${review_id}`)
```
Comment (Dutton1990) [inline src/components/review-list.jsx]: You can leave out the colon here! I think it looks cleaner in the url to just be reviews/2 etc.

File: src/components/review-card.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+                   <img src={currReview.review_img_url} alt={currReview.title} width="200"></img>
+                   <p>Category: {currReview.category}</p>
+                   <p>Designer: {currReview.designer}</p>
+                   <p>Owner: {currReview.owner}</p>
+                   <p>Created at: {(currReview.created_at).slice(0, -14)}</p>
+                   <p>Votes: {currReview.votes}</p>
+                   <p>{currReview.review_body}</p>
+               </div>
+        )}
+        </div>
```
Comment (Dutton1990) [inline src/components/review-card.jsx]: Be careful with using too many divs, I would change this outer one to be an empty fragment --> <> </>

**Comment by Dutton1990** [comment]
Just a few suggestions but looks great overall! Feel free to merge whenever you like!

**Pull request merged**